public/main.js

```diff
@@ -81,28 +82,42 @@ function reloadTodoList() {
         todoListPlaceholder.style.display = "none";
         todos.forEach(function(todo) {
             var listItem = document.createElement("li");
-            var compButton = document.createElement("button");
-            compButton.onclick = function(event) { completeTodo(todo.id, reloadTodoList); };
-            compButton.innerHTML = "Complete";
-            compButton.classList.add("completeButton");
-            compButton.setAttribute("data-id", todo.id.toString());
-            var delButton = document.createElement("button");
-            delButton.onclick = function(event) { deleteTodo(todo.id, reloadTodoList); };
-            delButton.innerHTML = "Delete";
-            delButton.classList.add("deleteButton");
-            delButton.setAttribute("data-id", todo.id.toString());
+            var completeButton = document.createElement("button");
```

Comment:
Looks like the setup for the completeButton and deleteButton has the same skeleton. This is fine for now, but if the same is required for another button, or the setup gets much longer think about splitting this out into its own function.